students/jingwan/users/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
+    user_name    = models.CharField(max_length=30, unique=True)
+    first_name   = models.CharField(max_length=161)
+    last_name    = models.CharField(max_length=11)
+    email        = models.CharField(max_length=254, unique=True)
+    password     = models.CharField(max_length=40)
+    phone_number = models.CharField(max_length=20, unique=True)
```

Comment:
- username은 붙여서 쓰는 것이 일반적입니다.
- 비밀번호는 다른 유저가 같은 것을 사용할 수도 있기 때문에 unique 속성은 제거해 주세요.

```suggestion
    username    = models.CharField(max_length=30, unique=True)
    first_name   = models.CharField(max_length=161)
    last_name    = models.CharField(max_length=11)
    email        = models.CharField(max_length=254, unique=True)
    password     = models.CharField(max_length=40)
    phone_number = models.CharField(max_length=20)
```

Reply:
마지막의 붙은 unique는 핸드폰번호입니다!

Reply:
그렇군요!! 착각 했습니다.